src/Components/Locations.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+import React, { Component } from "react";
+
+export default class Locations extends Component {
+  constructor() {
+    super();
+    this.state = {
+      locations: [],
+      isShown: false,
+    };
+  }
+  handleClick =()=>{
+      this.setState({
+       isShown: !this.state.isShown,
+      })
+  }
+  getLocations = async () => {
+    const { data } = await axios.get(
+      `https://ghibliapi.herokuapp.com/locations`
+    );
+    this.setState({
+      locations: data.map(
+        (loc) =>
+          (loc = { name: loc.name, climate: loc.climate, terrain: loc.terrain })
+      ),
+    });
+  };
+  componentDidMount() {
+    this.getLocations();
+  }
+  render() {
+    const { isShown, locations } = this.state;
+    const listItems = locations.map((loc, i) => (
+      <li style={{ color: "ivory" }} key={i}>
```

Comment:
Use loc.id as the key instead of using an array index! This is always preferable if possible!